docs/cmd/tkn_taskrun_delete.md

```diff
@@ -0,0 +1,46 @@
+## tkn taskrun delete
+
+Delete a taskrun in a namespace
+
+***Aliases**: rm*
+
+### Usage
+
+```
+tkn taskrun delete
+```
+
+### Synopsis
+
+Delete a taskrun in a namespace
+
+### Examples
+
+
+# Delete a TaskRun of name 'foo' in namespace 'bar'
+tkn taskrun delete foo -n bar
+
+tkn tr rm foo -n bar",
```

Comment:
Why there is a single `"`?

Reply:
oh I didn't realize it, thanks!!
I should correct it together with other documents, so how about making another PR after this??

Reply:
Cool.